### PR TITLE
HLT menu development for 14_0_X (2/N) [14_1_X]

### PR DESCRIPTION
**Pull request by mmusich**
#### PR description:

Development of HLT menus for CMSSW_14_0_X:
   * this is an intermediate update towards the 2024 proton-proton run data-taking.
   
This is the PR for the master branch, which will then be backported down to 14_0_X (the latter is the release cycle currently used for Run-3 HLT-menu development).

This update includes the integration of the following JIRA tickets:
- [CMSHLT-3050](https://its.cern.ch/jira/browse/CMSHLT-3050): [SMP] Remove unused L1 seeds from the W3Pi HLT path
- [CMSHLT-3056](https://its.cern.ch/jira/browse/CMSHLT-3056): [TSG/EXO] Remove `HLT_DiPhoton10smin*` paths from the menu
- [CMSHLT-3063](https://its.cern.ch/jira/browse/CMSHLT-3063): [BPH/EXO] Remove `ParkingDoubleElectronLowMass` stream from the GRun menu
- [CMSHLT-3064](https://its.cern.ch/jira/browse/CMSHLT-3064): [TAU] Remove deprecated L1T seeds from `hltL1sTauVeryBigOR`
- [CMSHLT-3024](https://its.cern.ch/jira/browse/CMSHLT-3024): [BTV/SUS] MSSM Hbb triggers migration from `DeepJet` to `PNet`
- [CMSHLT-3068](https://its.cern.ch/jira/browse/CMSHLT-3068): [Scouting] Removal of the `EG30` Scouting path
- [CMSHLT-3055](https://its.cern.ch/jira/browse/CMSHLT-3055): [PPS/ALCA] Configuration in HLT menu for new O2O/`LHCInfoPer*`
- [CMSHLT-3028](https://its.cern.ch/jira/browse/CMSHLT-3028): [B2G/HIG] Replacement of `MassSD` paths with `SoftDropMass`
- [CMSHLT-3033](https://its.cern.ch/jira/browse/CMSHLT-3033): [SUS] Update of `HLT_DoubleMu3_DCA_PFMET50_PFMHT60` and new auxiliary paths
- [CMSHLT-3029](https://its.cern.ch/jira/browse/CMSHLT-3029): [TAU] Adding DoubleTau+Jet paths including `PNet`
- [CMSHLT-3046](https://its.cern.ch/jira/browse/CMSHLT-3046): [TSG] Cleanup of MC Paths in `GRun`
- [CMSHLT-3037](https://its.cern.ch/jira/browse/CMSHLT-3037): [HIG/TAU/SMP] ParticleNet Versions of ETau, MuTau, and ETau Monitoring HLTs
- [CMSHLT-3031](https://its.cern.ch/jira/browse/CMSHLT-3031): [EXO] Integration of MDS+X triggers
- [CMSHLT-3077](https://its.cern.ch/jira/browse/CMSHLT-3077): [HIG/TAU] Fix IsoMu+PNet control triggers (`'ignore'` filters)
- [CMSHLT-3034](https://its.cern.ch/jira/browse/CMSHLT-3034): [EGM/JME/EXO/HIG] 50 GeV unprescaled single photon path
- [CMSHLT-3030](https://its.cern.ch/jira/browse/CMSHLT-3030): [PPS/SMP] Add PPS-Jet trigger to the 2024 menu
- [CMSHLT-3058](https://its.cern.ch/jira/browse/CMSHLT-3058): [HIG/EXO] VBF parking triggers for 2024
- [CMSHLT-3052](https://its.cern.ch/jira/browse/CMSHLT-3052): [HIG/TAU] VBF `PNet` HLT Integration
- [CMSHLT-3035](https://its.cern.ch/jira/browse/CMSHLT-3036): [L1T] HLT menu updates for `L1Menu_Collisions2024_v1_0_0`

In addition:
   * commit c7087749fa3306493afcc0ec267669524588ba7b : update of L1T `FULL`, `GRun`, `PIon`, `PRef`, `Special` menus in TSG GTs to `L1Menu_Collisions2024_v1_0_0_xml`
   * commit 5c3431c074777c3960ba9bd4cdbc2e786f6f8ca4 : percolates the [CMSHLT-3027](https://its.cern.ch/jira/browse/CMSHLT-3027) (already integrated in https://github.com/cms-sw/cmssw/pull/44023) changes to the `@Fake2` menu 
  
#### PR validation:

TSG integration tests.

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

Not a backport, but it will be backported to CMSSW_14_0_X.

----
## Important Note

**N.B.** Note to @cms-sw/orp-l2 this PR **MUST** be merged together with https://github.com/cms-sw/cmssw/pull/44078, that PR updates the `L1TUtmTriggerMenu` record to use `L1Menu_Collisions2024_v1_0_0_xml` in the `Configuration/AlCa` package: (see more details at this [cmsTalk post](https://cms-talk.web.cern.ch/t/update-of-the-2024-l1-menu-tag-l1menu-collisions2024-v1-1-0/35509)). Updates:
   * 2024 simulation `autoCond` keys
   * symbolic `run3_data_relval` and `run3_hlt_relval` keys

### CODE REVIEW

File: Configuration/AlCa/python/autoCond.py [hunk 1]
```diff
@@ -90,13 +90,13 @@
     # GlobalTag for MC production with realistic conditions for Phase1 2023 detector for Heavy Ion
     'phase1_2023_realistic_hi'     :    '140X_mcRun3_2023_realistic_HI_v3',
     # GlobalTag for MC production with perfectly aligned and calibrated detector for Phase1 2024
-    'phase1_2024_design'           :    '140X_mcRun3_2024_design_v1',
+    'phase1_2024_design'           :    '140X_mcRun3_2024_design_Candidate_2024_02_24_11_26_58',
```
Comment (mmusich) [inline Configuration/AlCa/python/autoCond.py:93]: @cms-sw/alca-l2 I created here GT candidates for testing purposes, please provide a final GT to be included here (and below).

**Comment by mmusich** [comment]
@cmsbuild, please test

**Comment by perrotta** [comment]
@mmusich the PR with the updated versioned GTs in autoCond is in https://github.com/cms-sw/cmssw/pull/44078
Please remove the corresponding files from this PR and retest the two PRs together afterwards

**Comment by mmusich** [comment]
> Please remove the corresponding files from this PR and retest the two PRs together afterwards

No need of two PRs, that risks only to create broken IBs if not merged together.

**Comment by perrotta** [comment]
> > Please remove the corresponding files from this PR and retest the two PRs together afterwards
> 
> No need of two PRs, that risks only to create broken IBs if not merged together.

If you don't mind I'd keep them separate, for AlCa boockeeping purpose, and insist with the release managers for having them merged together.
If you really prefere, you can remove the updates to autoCond here (they must be fixed anyhow) and import the very same commit that is in the other PR, so that once this is merged tge other will follow automatically.

File: Configuration/AlCa/python/autoCond.py (hunk 1 above)
Comment (mmusich) [reply Configuration/AlCa/python/autoCond.py]: Done in another PR https://github.com/cms-sw/cmssw/pull/44078

**Comment by mmusich** [comment]
test parameters:
   * pull_request = https://github.com/cms-sw/cmssw/pull/44078

**Comment by mmusich** [comment]
> If you don't mind I'd keep them separate, for AlCa boockeeping purpose, and insist with the release managers for having them merged together.

OK, let's hope for the best...

**Comment by mmusich** [comment]
@cmsbuild, please test

**Comment by mmusich** [comment]
+hlt
   * test pass and reveal expected changes;
   * additional empty DQM plots or [additional log lines](https://cmssdt.cern.ch/SDT/jenkins-artifacts/baseLineComparisons/CMSSW_14_1_X_2024-02-25-0000+00d3d0/61307/validateJR/logRootQA-events.log) are due to the fact that some DQM plugins create or remove plots based on the content of the menu because they select Paths based on regular expressions or similar. Other times, the name of the Paths are hardcoded in the DQM sequences, and POGs/PAGs need to update them themselves. TSG will follow-up with proponents after integration.

**Comment by rappoccio** [comment]
+1

**Pull request merged**